### PR TITLE
Reduce excessive RAM usage by using uint8 instead of float32

### PR DESCRIPTION
**Pull request by AnBucquet**
Related to [issue-110](https://github.com/neuropoly/ivado-medical-imaging/issues/110) and [PR-111](https://github.com/neuropoly/ivado-medical-imaging/pull/111). This PR aims to change GT and ROI type to `unint8` instead of `float32` to reduce excessive RAM usage.

### CODE REVIEW

**Review by olix86 — APPROVED** [review]
Nice thanks!

File: medicaltorch/transforms.py [hunk 1]
```diff
@@ -516,9 +516,10 @@ def sample_augment(self, input_data, params):
     def label_augment(self, gt_data, params):
         gt_data = self.sample_augment(gt_data, params)
         np_gt_data = np.array(gt_data)
-        np_gt_data[np_gt_data >= 0.5] = 1.0
+        np_gt_data[np_gt_data >= 0.5] = 255.0
```
Comment (charleygros) [inline medicaltorch/transforms.py:532]: good catch!

**Comment by charleygros** [comment]
What about those lines: https://github.com/neuropoly/medicaltorch/blob/master/medicaltorch/transforms.py#L395 ?

File: medicaltorch/transforms.py [hunk 2]
```diff
@@ -390,28 +399,31 @@ def __call__(self, sample):
         # TODO: Use the axis index for factoring this loop
         for x in range(input_data.shape[self.axis]):
             if self.axis == 0:
-                input_rotated[x,:,:] = F.rotate(Image.fromarray(input_data[x,:,:], mode='F'), angle)
+                input_rotated[x, :, :] = F.rotate(Image.fromarray(input_data[x, :, :], mode='F'), angle)
                 if self.labeled:
-                    gt_rotated[x,:,:] = F.rotate(Image.fromarray(gt_data[x,:,:], mode='F'), angle)
+                    gt_rotated[x, :, :] = F.rotate(Image.fromarray(gt_data[x, :, :], mode='L'), angle)
```
Comment (charleygros) [inline medicaltorch/transforms.py:404]: Not sure: No need to multiply by 255 before here?

Comment (AnBucquet) [reply medicaltorch/transforms.py:404]: Hum ... I don't think so as we use gt_data (which has been multiplied by 255 in `__getitem__`). But I might be wrong ?

**Pull request merged**